templates/CSI/cephfs/secret.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The same for secret

Reply:
@ebenahar the contents and fields of the secret.yaml are different for rbd and cephfs. So wouldn't it be better to  have distinct files for rbd and cephfs ?

Reply:
@nehaberry comparing these 2:
https://github.com/red-hat-storage/ocs-ci/blob/24b210fcd46e87dd8f8c2d1a5030f9a43280e865/templates/CSI/cephfs/secret.yaml
https://github.com/red-hat-storage/ocs-ci/blob/24b210fcd46e87dd8f8c2d1a5030f9a43280e865/templates/CSI/rbd/secret.yaml

Will rbd yaml always have kubernetes and admin and cephfs userID and userKey?